cmd/gopm/testdata/gopm-path-not-exists.txt

```diff
@@ -0,0 +1,19 @@
+! gopm -c gopmconfig --quit-delay 0 -p gopmextra/foo.cue
+stderr 'error:cannot load CUE path "gopmextra/foo.cue": path does not exist'
+
+-- gopmconfig/config.cue --
+package config
+
+config: grpc_server: {
+	network: "unix"
+	address: "gopm.socket"
+}
+
+-- gopmextra/options.cue --
+package config
```

Comment:
I'd be interested to know whether this package directive is needed. Kinda related: I'm wondering whether it might be nice to accept non-CUE files too e.g. JSON) so that it's easy to integrate with other kinds of data source.

Reply:
It is not required 👏🏻